CHANGES.md

```diff
@@ -1,66 +1,16 @@
-## current
+## v4.0.0 (2021-03-16)
```

Comment:
Should we keep this as current until we have a story for the new headers module?

Reply:
fixed